### PR TITLE
Task 79 start component

### PR DESCRIPTION
**Pull request by nghe0001**
# Description 📝
Shows the Start Again button

Screenshot from 2020-11-12 05-39-46

## Taiga Link 🔗

https://taiga.dts-stn.com/project/pond-dts-experimental-benefit-finder/us/79?milestone=36

## Checklist ✅
- [x] created story for all visual components
- [x] created or modified necessary unit tests
- [ ] created or modified e2e tests
- [ ] modified README or any applicable documentation on the changes you made

### CODE REVIEW

**Review by shewood — APPROVED** [review]
Looks good to me...

File: src/components/atoms/ActionButton.js [hunk 1]
```diff
@@ -19,13 +23,22 @@ export function ActionButton(props) {
       onMouseLeave={props.onMouseLeave}
       id={props.id}
     >
+      {props.icon}
```
Comment (Moro-Code) [inline src/components/atoms/ActionButton.js]: better to have the span element here if the icon class is passed in 

`{props.icon ? <span className={props.icon}/>: undefined} `

File: src/components/atoms/StartAgainButton.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ActionButton } from "../atoms/ActionButton";
+
+/**
+ * This component will add a start again button
+ */
+
+export function StartAgainButton(props) {
```
Comment (Moro-Code) [inline src/components/atoms/StartAgainButton.js]: hmmm not entirely sure what the point of this component is. perhaps modifying the ActionButton component is enough

**Review by Moro-Code — CHANGES_REQUESTED** [review]

File: src/components/atoms/StartAgainButton.js (hunk 2 above)
Comment (nghe0001) [reply src/components/atoms/StartAgainButton.js]: That's true , I will just modify the action button instead. Thanks Omar

File: src/components/atoms/ActionButton.js (hunk 1 above)
Comment (nghe0001) [reply src/components/atoms/ActionButton.js]: updated

**Pull request closed without merging**